samples/DatabaseManagement/app.js

```diff
@@ -1,89 +1,70 @@
 ﻿// @ts-check
-'use strict';
+"use strict";
 console.log();
-console.log('Azure Cosmos DB Node.js Samples');
-console.log('================================');
+console.log("Azure Cosmos DB Node.js Samples");
+console.log("================================");
 console.log();
-console.log('DATABASE MANAGEMENT');
-console.log('===================');
+console.log("DATABASE MANAGEMENT");
+console.log("===================");
 console.log();
 
-
-const cosmos = require('../../lib/');
+const cosmos = require("../../lib/");
 const CosmosClient = cosmos.CosmosClient;
-const config = require('../Shared/config')
-const databaseId = config.names.database
-  
+const config = require("../Shared/config");
+const databaseId = config.names.database;
+
 const endpoint = config.connection.endpoint;
 const masterKey = config.connection.authKey;
 
 // Establish a new instance of the CosmosClient to be used throughout this demo
-const client = new CosmosClient({endpoint, auth: { masterKey }});
+const client = new CosmosClient({ endpoint, auth: { masterKey } });
 
 //---------------------------------------------------------------------------------------------------
 // This demo performs the following CRUD operations on a Database
 //
-// 1. find Database  - Attempt to find a database by Id, if found then just complete the sample
-// 2. create Database    - If the database was not found, try create it
-// 3. read all Databases     - Once the database was created, list all the databases on the account
-// 4. read Database      - Read a database by its id (using new ID Based Routing)
-// 5. delete Database    - Delete a database given its id
+// 1. create Database    - If the database was not found, try create it
+// 2. read all Databases     - Once the database was created, list all the databases on the account
+// 3. read Database      - Read a database by its id (using new ID Based Routing)
```

Comment:
What's the new ID Based routing?

Reply:
name based, it's not new anymore. deleted.